src/core/server/src/workers/storage.rs

```diff
@@ -4,15 +4,18 @@
 
 use crate::*;
 
-counter!(
-    STORAGE_EVENT_LOOP,
-    "the number of times the event loop has run"
-);
-heatmap!(
-    STORAGE_QUEUE_DEPTH,
-    1_000_000,
-    "the distribution of the depth of the storage queue on each loop"
-);
+#[metric(
+    name = "storage_event_loop",
+    description = "the number of times the event loop has run"
+)]
+pub static STORAGE_EVENT_LOOP: Counter = Counter::new();
+
+#[metric(
+    name = "storage_queue_depth",
+    description = "the distribution of the depth of the storage queue on each loop"
+)]
+pub static STORAGE_QUEUE_DEPTH: Heatmap =
+    Heatmap::new(0, 8, 20, Duration::from_secs(60), Duration::from_secs(1));
```

Comment:
Are these the right parameters for the heatmap? Previously we seem to have replaced `heatmap!(100_000)` with `(0, 8, 20)` and this is 1M.

Reply:
Reducing to 0, 8, 17 which matches previous